ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,82 @@
+Hey, Thanks for reporting issues back to Nextcloud Face Recognition. Try filling this report with the greatest care so that we can help you. :smile:
```

Comment:
I would remove "greatest". It seems like we are some corporation:) It can kind-of be of putting. If I see this, I would say "screw you, I want to help you and you are intimidating me":)... but, maybe it is just me:)

Reply:
haha. Ok.. :sweat_smile:

Reply:
Please, try to complete this report in detail so we can help you easier. :smile:
¿?